applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_axisymmetric_integration_coefficients.cpp

```diff
@@ -0,0 +1,63 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Marjan Fathian
+//                   Gennady Markelov
+//
+
+#include "custom_elements/integration_coefficient_modifier_axisymmetric.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities/element_setup_utilities.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+using namespace std::string_literals;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(AxisymmetricIntegrationCoefficientsCalculator_ReturnsCorrectValue,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Set
+    const auto axisymmetric_integration_coefficient_calculator = IntegrationCoefficientsCalculator{
+        std::make_unique<IntegrationCoefficientModifierForAxisymmetricElement>()};
+    // The shape function values for this integration point are 0.2, 0.5 and 0.3 for nodes 1, 2 and 3 respectively
+    const Geometry<Node>::IntegrationPointType       integration_point(0.5, 0.3, 0.0, 0.5);
+    const Geometry<Node>::IntegrationPointsArrayType integration_points{integration_point};
+    Vector                                           detJs(1);
+    detJs <<= 2.0;
```

Comment:
We can make this variable `const`:
```suggestion
    const auto                                       detJs = Vector{ScalarVector{1, 2.0}};
```

Reply:
changed